lib/skate_web/router.ex

```diff
@@ -40,6 +40,10 @@ defmodule SkateWeb.Router do
     plug :put_secure_browser_headers
   end
 
+  scope "/docs", SkateWeb do
+    get "/agency-policies/AUP", Redirect, external: :aup
```

Comment:
**nitpick:** are Phoenix routers case-sensitive? If so, I think `aup` might be better than `AUP` so that we're not asking our users to type in a mixed-case URL.

Reply:
> are Phoenix routers case-sensitive

From what I can find, they are.


Lowercased in a80efb0d99057c856c6bf4de098a71e0ded741d2
We can revisit this some other time and add a "normalization plug" if we find that we need case insensitivity here https://elixirforum.com/t/how-to-match-uppercase-and-lowercase-route-name-in-phoenix/31037/10